### ISSUE TITLE
Cannot pass props to closeButton in tag

### ISSUE THREAD

**Issue by uraaaa24** [yamada-ui/yamada-ui#3014]
### Which package has an issue?

@yamada-ui/react

### Package Version

1.5.3

### Description

The current `Tag` component does not allow props to be passed to the internal closeButton.
For example, the `aria-label` of the closeButton contains the fixed value “Close tag”.
Fix this so that developers can pass props to the internal closeButton.

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

Chrome

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None
